lib/aes_256_cbc_encryptor.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class Aes256CbcEncryptor
```

Comment:
Was this built based on a reference implementation?

Reply:
So I had to write a script to be able to verify it with MHV a few weeks back. If you recall, they provided us with a sample hash and wanted it to work. So I tweaked a script I had locally until I got that to work. This is sort of a hybrid of that effort. I pulled from a lot of different examples on how to properly use the ruby library. The only thing we might want to look into more is the use of a "salt"